### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by JustinKuli**

### CODE REVIEW

File: test/integration/policy_info_metric_test.go [hunk 1]
```diff
@@ -161,7 +160,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
 			}
 
 			return status
-		}, defaultTimeoutSeconds, 1).Should(ContainSubstring("Unauthorized"))
```
Comment (yiraeChristineKim) [inline test/integration/policy_info_metric_test.go:164]: Why does this take longer?

**Comment by yiraeChristineKim** [comment]
Is this related to https://github.com/stolostron/governance-policy-framework-addon/actions/runs/6882031782/job/18720800096?pr=213?

File: test/integration/policy_info_metric_test.go [hunk 2]
```diff
@@ -161,7 +160,7 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policy_governance_info metric
...
+		}, "90s", 1).Should(ContainSubstring("Unauthorized"))
```
Comment (dhaiducek) [inline test/integration/policy_info_metric_test.go:163]: (optional) "90s" is probably clearer, but having a multiple follows conventions elsewhere...
```suggestion
		}, defaultTimeoutSeconds*3, 1).Should(ContainSubstring("Unauthorized"))
```

**Review by mprahl — APPROVED** [review]
This is good enough for now and we can update it to use the variable if we'd like.

**Pull request merged**